client/src/components/Header/styled.js

```diff
@@ -0,0 +1,68 @@
+import styled from "styled-components";
+
+export const HeaderLayout = styled.div`
+  .nav {
+    background-color: ${props => props.theme.color.lightpink};
+    border-bottom: 1px solid rgba(0, 0, 0, 0.0975);
+  }
+
+  .navHeader {
```

Comment:
A lot of css going into one styled component. It might be nicer to break them up a bit more? More granular components. I can show you an example tomorrow if you want?